x-pack/test/security_solution_cypress/es_archives/unmapped_fields/mappings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "index",
+  "value": {
+    "aliases": {
+      "mydata-7.12.0": {
+        "is_write_index": true
+      }
+    },
+    "index": "unmapped-7.12.0-2021.03.10-000001",
+    "mappings": {
+      "dynamic": false,
+      "properties":{
+        "@timestamp":{
+          "type":"date"
+        },
+        "mydestination":{
+          "properties":{
+            "ip":{
+              "index": "false",
+              "type":"keyword"
```

Comment:
I don't know if you want this to be keyword instead of ip? You should be able to use `ip` here and stay unmapped. Otherwise I would suggest maybe a different name? As a maintainer, seeing/using keyword for ip feels off.